cli/tools/registry/api.rs

```diff
@@ -0,0 +1,210 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
```

Comment:
Are these new code changes in `registry/` intentional? Looks like code from `publish/`

Reply:
Bad rebase, oops (for some reason when I rebased, it said there were no conflicts, and just added back these deleted files)